packages/data-grid/src/quickFilter/QuickFilter.jsx

```diff
@@ -30,11 +30,28 @@ const propTypes = {
 };
 
 function QuickFilter({ filters, onFilterSelect }) {
+  /** Used for the options array when there is no items to filter by */
+  const emptyFilterOption = [
+    {
+      label: 'No items to filter by',
+      disabled: true,
+    },
+  ];
+
   return (
     <div className="dg-quick-filter-container">
       <div className="dg-quick-filter-filters">
         {filters.map((filter, idx) => {
-          return (
+          return !filter.options ? (
+            <ButtonSelect
+              className="dg-quick-filter-empty"
```

Comment:
we can do condition logic for classnames to using an npm package called classnames https://www.npmjs.com/package/classnames